virt/release_notes/virt-4-19-release-notes.adoc

```diff
@@ -112,6 +112,24 @@ As a cluster administrator, you can prevent users from enabling VM delete protec
 ** `HighCPUWorkload`
 ** `NodeNetworkInterfaceDown`
 
+//CNV-50346
+* New metrics are now available and improve the observability of virtual machines (VMs) and virtual machine instances (VMIs). You can use these metrics to monitor the following VM lifecycle events, resource usage, and migration details:
++
+--
+** Migration metrics
+** vNIC networking information metrics
+** Allocated storage size metrics for running and stopped VMs
+--
++
+In addition, the following VM and VMI metadata metrics are now available:
++
+--
+** The `pod_name` label in `kubevirt_vmi_info`
+** UID in VM and VMI metrics
+** The VM creation date
+--
++
+For a complete list of virtualization metrics, see link:https://github.com/kubevirt/monitoring/blob/main/docs/metrics.md[KubeVirt components metrics].
```

Comment:
not a merge-blocker, but starting this section with bullets and then not have bullets on the last two items looks funky in the render